pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-self-validation.conf

```diff
@@ -0,0 +1,149 @@
+{
+  "validations": [
+    {
+      "messageTemplate": {
+        "templateType": "HTML",
+        "templateText": "$singular_consent_self_validation",
+        "variables": [
+          {
+            "name": "singular_consent_self_validation",
+            "translations": [
+              {
+                "language": "en",
+                "text": """Please check the date of birth you have entered. If the date is correct, then you
+                               are younger than the age required by local law to self-enroll in this study. However,
+                               you may ask your parent or guardian to enroll you in the study. If you think there
+                               is a mistake, please reach out to us at (650) 561-6750 or
+                               <a href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>.
+                            """
+              }
+            ]
+          }
+        ]
+      },
+      "stableIds": ["PREQUAL_AGE", "CONSENT_SELF_DATE_OF_BIRTH", "PREQUAL_COUNTRY", "PREQUAL_STATE", "PREQUAL_PROVINCE"],
+      "precondition": """
```

Comment:
I believe its cleaner if we can move these long pex into substitutions.conf file.
Please check "_pex": {} in substitutions.conf and we might already have these pex in there, if not add there and substitute here.

Reply:
I'm not sure about that. To me it just complicates things. Besides after yesterday fixes expressions doesn't look so heavy